demos/ag-alert-keyvault-advisor/azuredeploy.json

```diff
@@ -0,0 +1,64 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "alertName":{
+         "type":"String"
+      },
+      "applicationGatewayResourceGroup":{

```

Comment:
change this to prompt only for the resourceGroup name (and sub if needed) and create the resourceId in the template

Reply:
As an MVP for these quick deployment of Azure Monitor alerts, we have taken this design decision to let customers provide the ResourceIDs manually (for both - an existing Application Gateway and Action Group). We can refine this experience in the further iterations. The same experience is used for the already published https://github.com/Azure/azure-quickstart-templates/blob/master/demos/ag-alert-backend-5xx/README.md